packages/block-editor/src/components/block-navigation/appender.js

```diff
@@ -48,7 +48,7 @@ export default function BlockNavigationAppender( {
 							isLastRow={ position === rowCount }
 							terminatedLevels={ terminatedLevels }
 						/>
-						<ButtonBlockAppender
+						<Inserter
```

Comment:
I don't think `Inserter` handles the `TreeGridCell` props (ref, onFocus, tabIndex) correctly. 

Perhaps it'd be clearer if they were specified individually instead of spread as this mistake has already happened in other PRs as well.

Reply:
Right you are, and good idea—it's easy to miss for folks doing find-and-replace style programming like I am in this PR 😛

Reply:
FaR programming is the future!

Reply:
Fixed in 742606d1d8, but I'm not sure how to test it?

Reply:
Yeah, it's a bit tricky as there are other things broken in the navigation. The props provide the Roving Tab Index behavior. It should work a bit like this:
1. On the nav menu screen press tab until focus is within block navigation.
2. Use the arrow keys to move focus around block navigation.
3. Observe that arrow keys move focus around the navigation in a grid-like manner
4. Press shift+tab, focus should leave block navigation (block navigation has a single tab stop)
5. Press tab, focus should return to the last focused element in block navigation.